README.md

```diff
@@ -86,7 +88,15 @@ Navigate to the [chrome://extensions](//chrome://extensions) page, enable "Devel
 
 If you just want to try out and debug the extension, go to [about:debugging#addons](//about:debugging#addons). Then press "Load Temporary Add-On" and select the built `manifest.json` from the `dist/firefox` extension directory.
 
-If you want to install this addon permanently, go to [about:addons](//about:addons) and click on the small cog icon. Select `Install Add-on From File...` and choose `dist/firefox`.
+If you want to install the extension permanently, you have two options:
+
+1. Install a self-signed add-on
+
+  Get a `tickety_tick-<version>.xpi` file from the [releases](https://github.com/bitcrowd/tickety-tick/releases) or [signing by a maintainer](./MAINTAINERS.md#sign-add-on-for-firefox). Then go to [about:addons](//about:addons), click on the small cog icon, select `Install Add-on From File...` and choose your `.xpi` file.
```

Comment:
We don't offer the `.xpi` files in the releases yet, so the documentation might mislead people in the meantime? Also, this doesn't really help me if I want to permanently install a custom-built version myself right? I'd still only be able to choose from the official releases (unless I'm a Tickety-Tick maintainer with access to the credentials).

So all in all, it seems to me like this is not really an option and everyone who wanted to tweak something about Tickety-Tick and use their own version can only use the 2. option?

🤔

Reply:
True for "tweak something and use their own version" but not for "I need to downgrade because the current version does not work for me" - which is a valid and in my opinion also very likely use case.

Mozilla's add-ons don't allow users to actually "control" which version they are using. Some people even stop semver-ing their extensions and instead use current date for the version 😂.

From what I see, installing from file is the only option to downgrade then 🤔 .

Using Chrome, I can download all old `.xps` files, so we could backfill at least the last two versions in the releases before moving to version 4 😉. Just to play safe.

Reply:
Then let's rather create a separate section for "Installing an older version" and move the hint about the `xpi`/`xps` files there?

The "Install a custom-build version" section can then just focus on the "Enable unsigned add-ons" part.

I think this'd be less confusing. Depending on what I want to achieve, I can get clear separate instructions like this.

🙂

Reply:
Makes sense 👍

Reply:
Moved it to a separate optional section in the installation instructions:

<img width="652" alt="Screenshot 2020-05-07 at 10 43 44" src="https://user-images.githubusercontent.com/3491815/81273852-c929d700-904f-11ea-9ea3-e26ac962faa9.png">